app/sidekiq/lighthouse/poll_form526_pdf.rb

```diff
@@ -76,6 +76,20 @@ def perform(submission_id) # rubocop:disable Metrics/MethodLength
       Sentry.set_tags(source: '526EZ-all-claims')
 
       with_tracking('Form526 Submission', submission.saved_claim_id, submission.id, submission.bdd?) do
+        # Check the submission.created_at date, if it's more than 2 days old and the PDF is missing
```

Comment:
I think we want this check further down in the method. We don't want to set the job status to `pdf_not_found` _before_ we even check whether or not the PDF exists. I was thinking more in the else condition, where normally we would "keep on retrying" but in _this_ case- don't. It's not there and it's older than 48 hours.

Reply:
Oh I see, I was understanding the ask as "exit before doing anything meaningful if we already know we're past the limit"

Got it, will make the adjustment